api/applyEffects.php

```diff
@@ -271,6 +271,69 @@
                 throw new \Exception('Failed to copy photo.');
             }
         }
+
+        // Apply rembg if enabled
```

Comment:
Should be handled earlier, after rotating, but before applying other effects like text, Frame etc. Recreation of thumbnail should not be needed.